aostypes/aostypes.go

```diff
@@ -180,16 +200,78 @@ func (d *Duration) UnmarshalJSON(b []byte) (err error) {
 		return nil
 
 	case string:
-		duration, err := time.ParseDuration(value)
-		if err != nil {
-			return aoserrors.Wrap(err)
-		}
+		if !strings.Contains(value, "P") {
```

Comment:
Instead of checkin `P` better to do as following:
```go
duration, err := parseISO8601Duration(value)
if err == nil {
    d.Duration = duration
    break
}

duration, err := time.ParseDuration(value)
if err == nil {
    d.Duration = duration
    break
}

return aoserrors.New("can't parse duration")
```